style.css

```diff
@@ -0,0 +1,95 @@
+body {
+    background:url(tile\ -\ לוח\ שעם.jpg);
+    font-family: Arial, sans-serif;
+  }
+ 
+  h1 {
+    text-align: center;
+    font-family: 'Courier New', Courier, monospace;
+    margin-top: 20px;
+  }
+ 
+  body form {
+    background: repeating-linear-gradient(white, white 28px, #cce5ff 29px, white 30px);
+    border-left: 5px solid red;
+    padding: 20px;
+    max-width: 500px;
+    margin: 20px auto;
+    border-radius: 10px;
+    box-shadow: 0 0 8px #aaa;
+  }
+ 
+  body #task-board {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(260px, 1fr));
+    gap: 20px;
+    padding: 20px;
+    max-width: 1200px;
+    margin: auto;
+    direction: ltr;
+  }
+ 
+  .task-card {
+    background:url(notebg\ -\ פתק\ צהוב.png);
+    width: 70%;
+    height: 220px;
+    padding: 15px;
+    border-radius: 10px;
+    box-shadow: 2px 2px 6px rgba(0, 0, 0, 0.2);
+    position: relative;
+    overflow-y: auto;
+    opacity: 0;
+    transition: opacity 0.5s ease-in;
+  }
+ 
+  body #task-board .task-card.show {
+    opacity: 1;
+  }
+ 
+  body #task-board .task-card:hover .delete-btn {
+    display: block;
+    padding: 15px;
+  }
+ 
+  body #task-board .task-card p {
+    margin: 0;
+    overflow-y: auto;
+    height: 130px;
+  }
+ 
+  body #task-board .task-card small {
+    display: block;
+    margin-top: 10px;
+    text-align: left;
+    font-size: 0.85em;
+    color: #555;
+  }
+ 
+  body #task-board .task-card .delete-btn {
+    position: absolute;
+    top: 5px;
+    right: 10px;
+    display: none;
+    cursor: pointer;
+    color: red;
+    font-size: 18px;
+  }
+  @media (max-width: 600px) {
+  body form {
+    max-width: 90%;
+    padding: 15px;
+  }
+
+  #task-board {
+    grid-template-columns: 1fr; 
+    padding: 10px;
+  }
+
+  .task-card {
```

Comment:
doesn't look so good with plenty of text:
<img width="431" height="733" alt="Screenshot 2025-08-04 at 11 48 57" src="https://github.com/user-attachments/assets/ff4b6c92-4870-4781-a54e-613af00b375b" />